modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetLayoutDisplayPageObjectProviderCheck.testjava

```diff
@@ -61,5 +61,6 @@ public class UpgradeJavaGetLayoutDisplayPageObjectProviderCheck {
 	private final LayoutDisplayPageProviderRegistry
 		_layoutDisplayPageProviderRegistry;
 	private final Portal _portal;
+	private LayoutDisplayPageProvider layoutDisplayPageProvider;
```

Comment:
@TamyrisTorres, can you please clarify why you're adding the same content here and in the file below?

Reply:
@natocesarrego, This check only works because of the bug, so we need to change it.

In this test case, in the second method, we need to validate the "layoutDisplayPageProvider" variable, but we cannot put the same case as the first method. So, I put it as a java variable.